haiku-apps/genio/genio_extension_showdocumentation-1.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="'Show Documentation' extension for Genio The Haiku IDE"
+DESCRIPTION="Experimental 'Show Documentation' extension for Genio The Haiku IDE."
+COPYRIGHT="2024-2025 The Genio Team"
+HOMEPAGE="https://github.com/Genio-The-Haiku-IDE/Genio/releases"
+LICENSE="BSD (3-clause)
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Genio-The-Haiku-IDE/genio-extension-showdocumentation/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="b87a1b310f3ecaac229c1d559a02f9fb51ad7c7ac422b6fb2345ffed99c9589e"
+SOURCE_FILENAME="genio-extension-showdocumentation-v$portVersion.tar.gz"
+SOURCE_DIR="genio-extension-showdocumentation-$portVersion"
+
```

Comment:
You could grab the archive for the docset like this:
```
SOURCE_URI_2="https://github.com/Genio-The-Haiku-IDE/org.haiku.HaikuBook.docset/archive/9c12e19a189395c99fb9c7dc10eab539ffef37dc.tar.gz"
CHECKSUM_SHA256_2="7263683520f6ecf176905a7a73a39ef174c8174df5f139e321037caaed965952"
```

Reply:
> You could grab the archive for the docset like this:
> 
> ```
> SOURCE_URI_2="https://github.com/Genio-The-Haiku-IDE/org.haiku.HaikuBook.docset/archive/9c12e19a189395c99fb9c7dc10eab539ffef37dc.tar.gz"
> CHECKSUM_SHA256_2="7263683520f6ecf176905a7a73a39ef174c8174df5f139e321037caaed965952"
> ```

Wouldn't this grab the archive at package building time ?
I'd like to grab it on installation, not on building.